lib/route.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
-  mounted () {
-    window.location.reload()
+  layout: 'public',
+  authenticated: false,
+  middleware () {
```

Comment:
Middleware receives `{ redirect }` which probably would be safer to use than `window.location`

Reply:
It should be what was there before so it's backwards compatible, I just forgot to remove it ;)